client/views/windows/settings.tpl

```diff
@@ -14,6 +14,10 @@
 	</div>
 
 	<div class="row">
+		<div class="col-sm-12">
+			<button type="button" class="btn" id="make-default-client">Open <code>irc://</code> URLs with The Lounge</button>
```

Comment:
Since GitHub hides the comment: https://github.com/thelounge/thelounge/pull/2571#discussion_r197602539